<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1576</width>
    <height>980</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QFrame" name="frame_3">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>0</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item>
            <widget class="QTabWidget" name="tabWidget_3">
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_15">
              <attribute name="title">
               <string>synoptic viewer</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_16">
               <item>
                <layout class="QGridLayout" name="gridLayout_14">
                 <item row="0" column="4">
                  <widget class="QPushButton" name="pushButton_render">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="text">
                    <string>Render Viewer</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="comboBox_viewer_filename">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>0</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="3">
                  <widget class="QComboBox" name="comboBox_viewer_obj_name">
                   <property name="minimumSize">
                    <size>
                     <width>100</width>
                     <height>0</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="2">
                  <widget class="QLabel" name="label_80">
                   <property name="text">
                    <string>Viewer objects</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_79">
                   <property name="text">
                    <string>Viewer files</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="5">
                  <spacer name="horizontalSpacer_8">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="beamlineSynopticViewer" name="widget_synoptic" native="true">
                 <property name="mouseTracking">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3">
                 <item>
                  <widget class="QLabel" name="label_19">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="text">
                    <string>scale viewer by 0.5 up to 2.5</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSlider" name="horizontalSlider_sf">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>5</number>
                   </property>
                   <property name="value">
                    <number>2</number>
                   </property>
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="tickPosition">
                    <enum>QSlider::NoTicks</enum>
                   </property>
                   <property name="tickInterval">
                    <number>0</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QFrame" name="frame_2">
          <property name="maximumSize">
           <size>
            <width>9000</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="font">
              <font>
               <family>Arial</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tab_13">
              <attribute name="title">
               <string>Spock</string>
              </attribute>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QtSpockWidget" name="widget_spock" native="true">
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_4">
              <attribute name="title">
               <string>Console</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="TerminalWidget" name="widget_terminal" native="true"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1576</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TerminalWidget</class>
   <extends>QWidget</extends>
   <header>smart.gui.widgets.terminal_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtSpockWidget</class>
   <extends>QWidget</extends>
   <header>sardana.taurus.qt.qtgui.extra_sardana.qtspock.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>beamlineSynopticViewer</class>
   <extends>QWidget</extends>
   <header>smart.gui.widgets.synoptic_widgets.beamline_viewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
